backend/build.gradle

```diff
@@ -31,6 +31,7 @@ dependencies {
     implementation 'org.liquibase:liquibase-core'
     runtimeOnly 'org.postgresql:postgresql'
     implementation 'com.googlecode.libphonenumber:libphonenumber:8.12.15'
```

Comment:
These  two and spring-boot-starter-validation should be grouped together as a `// data validation` section, methinks.